Src/BootCamp.Chapter/FileCleaner.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class FileCleaner
+    {
+        private const char DollarSign = '£';
+
+        /// <summary>
+        /// Cleans up dirtyFileName 
+        /// </summary>
+        /// <param name="dirtyFile">Dirty file with "_" placed in random places.</param>
+        /// <param name="cleanedFile">Cleaned up file without any "_".</param>
+        public static void Clean(string dirtyFile, string cleanedFile)
+        {
+            var corruptedBalanceLines = File.ReadAllLines(dirtyFile);
```

Comment:
If file is null, it will throw. You need to check if the path is not null before you can read.

Reply:
I think, I understand.  I have made code changes to it.